### DISCUSSION TITLE
Add links to guest speaker background info

### DISCUSSION THREAD

**Discussion by rubenmak** [OpenMined/courses#149] [Feedback] [answered]
L2C8: couldn't easily find Say Mine on Google because I misspelled it as Stay Mine. Suggestion: maybe it would be nice to add a link to the project in the bottom section 'recommend reading'? In general I think it would be great to have links to guest speaker background information below the videos they appear in!

**Comment by em-blue** [comment, marked as answer]
@rubenmak Good idea - fixed! Just added https://saymine.com/ in the additional resources of that video :-) 
We're working on getting recommended reading on all the videos - people just need some holiday first to rest. Feel free to suggest certain articles you find related to the material btw!

**Reply by em-blue** [reply]
Hey @rubenmak, you should see resources below most videos now!

**Comment by rubenmak** [comment]
Awesome!

I was keeping track of what quest speakers appeared in each video to give
suggestions, but you already got it covered! :)
